docs/detections/alerts-view-details.asciidoc

```diff
@@ -6,111 +6,198 @@
+
+* **Fields enriched with threat intelligence**: Shows the number of matched indicators that are present on an alert that _wasn't_ generated from an indicator match rule. If none exist, the total number of matched indicators is zero.
 
 [discrete]
-[[alerts-enrich-ti]]
-==== Alerts enriched with threat intelligence
-Alerts can be enriched with contextually relevant threat intelligence that you can use for triaging and investigating alerts. If available, this information appears in the Enriched data section.
+[[expanded-threat-intel-view]]
+==== Expanded threat intelligence view
+
+From the right panel, click **Threat intelligence** to open the expanded Threat intelligence view within the left panel. 
+
+NOTE: The expanded threat intelligence view queries indices specified in the `securitySolution:defaultThreatIndex` advanced setting. Refer to <<update-threat-intel-indices, Update default Elastic Security threat intelligence indices>> to learn more about threat intelligence indices. 
+
+[role="screenshot"]
+image::images/expanded-threat-intelligence-view.png[Expanded view of threat intelligence on the alert, 70%]
+
+The expanded Threat intelligence view shows individual indicators within the alert document. You can expand and collapse indicator details by clicking the arrow button at the end of the indicator label. Each indicator is labeled with values from the `matched.field` and `matched.atomic` fields and displays the threat intelligence provider. 
 
-When gathering threat intelligence data for an alert, {elastic-sec} queries the alert for indicator matches from the past 30 days. The query looks for the following fields:
+Matched threats are organized into two sections, described below. Within each section, matched threats are shown in reverse chronological order, with the most recent at the top. All mapped fields are displayed for each matched threat.
+
+**Threat match detected**
+
+The Threat match detected section is only populated with indicator match details if you're examining an alert that was generated from an indicator match rule. Indicator matches occur when alert field values match with threat intelligence data you've ingested. 
+
+**Fields enriched with threat intelligence**
+
+Threat intelligence can also be found on alerts that weren't generated from indicator match rules. To find this information, {elastic-sec} queries alert documents from the past 30 days and searches for fields that contain known threat intelligence. If any are found, they're logged in this section. 
+
+{elastic-sec} searches the alert document for the following fields:
+
+NOTE: Additional threat intelligence data is generated for indicator match alerts when indicators are matched with source events, as outlined by the conditions in the relevant indicator match rule. 
```

Comment:
Spoke with @nkhristinin on this note and, with this rewrite, it no longer seems necessary. 

```suggestion
```